### ISSUE TITLE
Video of animated layers

### ISSUE THREAD

**Issue by kbailey-noaa** [asascience-open/oceansmap#32]
Add capability for a user to create a video out of selected, animated layers.

**Comment by kbailey-noaa** [comment]
@kknee whatever happened to this?  I thought this enhancement was made but then there was an issue so it was rolled back...

**Comment by Bobfrat** [comment]
@kbailey-noaa I've deployed the service to the dev server. Please have a look:
http://dev.oceansmap.com/eds/

**Comment by kbailey-noaa** [comment]
@Bobfrat Thanks! I was able to download an MP4.  I tried surface currents (HFR) and CNAPS water temp and found the water temp covers the current vectors in the video (but in the map it looks good).

**Comment by Bobfrat** [comment]
@kbailey-noaa the animation setup modal puts the layer order onus on the user. The first layer in the list draws on top. You can use the up and down arrows to reorder your layers. We can think about how to make this feature more obvious to the user.

![Screen Shot 2019-09-05 at 10 39 17 AM](https://user-images.githubusercontent.com/5702672/64352354-e2aee680-cfc9-11e9-89a0-c80c95bf7f1f.png)

**Comment by kbailey-noaa** [comment]
@Bobfrat Thanks!  That worked.

Is the picture expected to be this coarse, or is it just my system?
![image](https://user-images.githubusercontent.com/11823385/64353064-1a6a5e00-cfcb-11e9-8801-9b73d8cdd840.png)


vs

![image](https://user-images.githubusercontent.com/11823385/64353191-556c9180-cfcb-11e9-8bf3-8f958285a4fb.png)

**Comment by kbailey-noaa** [comment]
@Bobfrat If image quality is as expected then I can close this issue and we can put into production.

**Comment by Bobfrat** [comment]
@kbailey-noaa yes, this is done server side so this is the resolution I'm getting as well. We can open another issue if you'd like to improve on it.
